.github/workflows/no-text.yml

```diff
@@ -0,0 +1,32 @@
+name: no-text
+on:
+  - push
+  - pull_request
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: ['ubuntu-latest']
+        ghc: ['latest']
```

Comment:
We might want better coverage of GHC versions, but we can add that if it turns out that the compatibility story is not as simple as we hope.

Reply:
I've added 7.10 to the matrix (this is the earliest available through `ghcup`) and checked 7.4 manually. This actually unveiled some issues (e. g., old `base` does not have `Data.List.uncons`), now fixed.